client/src/styles/legacy/_mixins.scss

```diff
@@ -119,19 +42,21 @@ Used in side panels and action tabs
 	}
 
 	.field {
-		/*
+		/**
 		 * Fields are more compressed in some areas compared to the
 		 * main content editing window so the below alters the internal
 		 * spacing of the fields so we can move that spacing to between
 		 * the form fields rather than padding
 		 */
 		border-bottom:none;
-		box-shadow: none;
+    box-shadow: none;
```

Comment:
Wrong indentation? Go through the pull request again please and ensure we're following our coding conventions there (at least in terms of indentation).

Reply:
@chillu Editor config says `indent_style = space`. The indentation should use spaces instead but I don't want to replace tabs with spaces everywhere.

Reply:
@chillu I've places tabs with 2 spaces for indentation now